### DISCUSSION TITLE
Arduino Giga display on Giga R1

### DISCUSSION THREAD

**Discussion by mikedalli** [Bodmer/TFT_eSPI#3248] [Compatible displays and setup files]
Just got an Arduino giga display and it is mounted directly on a Giga R1 and was hoping I can program it with the TFT_eSPI library. Is this display even compatible with the library?

**Comment by MasonMenzies** [comment]
Did you ever get this display setup? I'm also hoping to get mine working with TFT_eSPI and LVGL.
